app/src/main/java/com/infomaniak/drive/ui/addFiles/ImportFilesDialog.kt

```diff
@@ -88,20 +86,25 @@ class ImportFilesDialog : DialogFragment() {
 
     private suspend fun importFiles() {
         var errorCount = 0
+        val uploadFilesJobs = mutableListOf<Job>()
         navArgs.uris.forEach { uri ->
-            runCatching {
-                initUpload(uri)
-            }.onFailure {
-                it.printStackTrace()
-                Sentry.withScope { scope ->
-                    scope.level = SentryLevel.ERROR
-                    scope.setExtra("uri", uri.toString())
-                    Sentry.captureException(it)
+            uploadFilesJobs.add(lifecycleScope.launch {
```

Comment:
I don't really get what it has too do with the initial problem ? 
Won't it only be an optimization by making this in separate coroutines ?